Description: Fix taking snapshots of '/' itself
Bug: https://github.com/kdave/btrfs-progs/issues/162
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=920886

--- a/utils.c
+++ b/utils.c
@@ -2079,7 +2079,7 @@ int find_mount_root(const char *path, ch
 	while ((ent = getmntent(mnttab))) {
 		len = strlen(ent->mnt_dir);
 		if (strncmp(ent->mnt_dir, path, len) == 0 &&
-		    (path[len] == '/' || path[len] == '\0')) {
+		    (len == 1 || path[len] == '/' || path[len] == '\0')) {
 			/* match found and use the latest match */
 			if (longest_matchlen <= len) {
 				free(longest_match);
