Description: Use debian install layout
Author: Dimitri John Ledkov <xnox@ubuntu.com>
Forwarded: not-needed
Last-Update: 2018-05-14

--- btrfs-progs-4.16.1.orig/Makefile
+++ btrfs-progs-4.16.1/Makefile
@@ -651,7 +651,7 @@ endif
 ifeq ($(PYTHON_BINDINGS),1)
 install_python: libbtrfsutil_python
 	$(Q)cd libbtrfsutil/python; \
-		$(PYTHON) setup.py install --skip-build $(if $(DESTDIR),--root $(DESTDIR)) --prefix $(prefix)
+		$(PYTHON) setup.py install --install-layout=deb --skip-build $(if $(DESTDIR),--root $(DESTDIR)) --prefix $(prefix)
 
 .PHONY: install_python
 endif
